packages/service-worker/src/services/appcheck/utils/sendRequest.ts

```diff
@@ -0,0 +1,34 @@
+import { isDevelopment } from '@core/common';
+
+type Params = {
+  path: string;
+  payload: Record<string, unknown>;
+  timeout?: number;
+};
+
+export const sendRequest = async ({ path, payload, timeout }: Params) => {
+  const url = process.env.ID_SERVICE_URL;
+
+  if (!url) {
+    throw new Error('ID_SERVICE_URL is missing');
+  }
+
+  const manifestResponse = await fetch(chrome.runtime.getURL('manifest.json'));
+  const manifestBuffer = await manifestResponse.arrayBuffer();
+  const encodedManifest = Buffer.from(manifestBuffer).toString('base64');
```

Comment:
Out of curiosity, why do we need to `fetch()` the manifest here instead of using `runtime.getManifest()`?
Is it to ensure the exact same data is pulled as during the release workflow?

If so, can we add some comments about it so that future generations don't break it thinking this should be the same? :D